src/actions/integrations.ts

```diff
@@ -53,17 +47,11 @@ export function getIncomingHooks(teamId = '', page = 0, perPage: number = Genera
 
 export function removeIncomingHook(hookId: string): ActionFunc {
     return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
-        dispatch({type: IntegrationTypes.DELETE_INCOMING_HOOK_REQUEST, data: {}}, getState);
-
         try {
             await Client4.removeIncomingWebhook(hookId);
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch, getState);
 
-            dispatch(batchActions([
-                {type: IntegrationTypes.DELETE_INCOMING_HOOK_FAILURE, error},
-                logError(error),
```

Comment:
@nadalfederer Hey, We still like to dispatch the log errors here. Would you mind adding them back?

Reply:
Thanks for the review. I fixed it